docs/source/guide/ml_create.md

```diff
@@ -100,17 +114,17 @@ The `parsed_label_config` looks like the following:
 ```
 If you use an alias for the [Label tag](/tags/label.html), the `labels` dictionary contains the label aliases. Otherwise, it lists the label values.
 
-## Make predictions with your ML backend
+### Make predictions with your ML backend
 
-Create an ML backend with an inference call to get predictions from your model on-the-fly while annotating. 
+Create an ML backend with an inference call to get predictions from your model quickly while annotating. 
```

Comment:
there should be "on-the-fly", it means that LS asks ML backend to geneate predictions when a user opens a task